CHANGES.rst

```diff
@@ -22,6 +22,7 @@ Bugfixes:
 
 * Make KafkaStorageError retriable after metadata refresh like in other
   implementations (pr #1115 by @omerhadari)
+*  Handle node failure properly in sender (pr #1135 by @vmaurin)
```

Comment:
```suggestion
* Handle node failure properly in sender (pr #1135 by @vmaurin)
```
\+ could you please describe in short the problem it fixed? Hanging when coordinator is gone?